src/GraphQL.Tests/Types/AutoRegisteringInputObjectGraphTypeTests.cs

```diff
@@ -238,6 +247,31 @@ public void CanOverrideFieldList()
             graph.Fields.Find("Field1").ShouldNotBeNull();
         }
 
+        [Fact]
+        public void CanAddFieldInfos()
+        {
+            var graph = new TestFieldSupport<TestClass>();
+            graph.Fields.Find("Field5").ShouldNotBeNull();
+        }
+
+        [Fact]
+        public void DoesParse()
+        {
+            var dic = new Dictionary<string, object?>()
+            {
+                { "Field1", 11 },
+                { "Field3", 13 },
+                { "Field5", 15 },
+                { "Field6AltName", 16 },
+            };
+            var graph = new TestFieldSupport<TestClass>();
+            var actual = graph.ParseDictionary(dic).ShouldBeOfType<TestClass>();
+            actual.Field1.ShouldBe(11);
+            actual.Field3Value.ShouldBe(13);
```

Comment:
13 ? How does Field3 translates into Field3Value?

Reply:
Field3 is a write-only property which contains code to populate Field3Value for purposes of testing only.

Reply:
🤦‍♂️ I see